app/models/conversation.rb

```diff
@@ -210,6 +210,8 @@ def notify_status_change
   end
 
   def dispatcher_dispatch(event_name)
+    return if Current.user.nil?
```

Comment:
Is there a specific reason for us to require Current.user ?

This can have unintended consequences.  At the moment we are sending webhook events for events like `conversation_resolved` when the system auto resolves a conversation where `Current.user` won't be present. This change will break that behaviour.